packages/matchbox/src/components/Page/Page.js

```diff
@@ -1,149 +1,218 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-
-import { buttonFrom } from '../Button';
-import { ChevronLeft } from '@sparkpost/matchbox-icons';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+import { pick } from '@styled-system/props';
+import { createPropTypes } from '@styled-system/prop-types';
+import { ChevronLeft, MoreHoriz } from '@sparkpost/matchbox-icons';
+import { ActionList } from '../ActionList';
+import { Box } from '../Box';
+import { Button } from '../Button';
 import { EmptyState } from '../EmptyState';
-import { UnstyledLink, linkFrom } from '../UnstyledLink';
+import { Popover } from '../Popover';
+import { UnstyledLink } from '../UnstyledLink';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
 import { filterByVisible } from '../../helpers/array';
 
-import styles from './Page.module.scss';
+const Wrapper = styled('div')`
+  ${margin}
+`;
+
+function Breadcrumb({ content, ...rest }) {
+  return (
+    <UnstyledLink {...rest} fontSize="200" lineHeight="200" fontWeight="medium">
+      <Box as="span" display="inline-flex" align-items="center">
+        <ChevronLeft size={20} />
+        {content}
+      </Box>
+    </UnstyledLink>
+  );
+}
+
+function Subtitle({ subtitle }) {
+  if (!subtitle) {
+    return null;
+  }
+
+  if (typeof subtitle === 'string') {
+    return (
+      <Box
+        as="h2"
+        fontSize={['300', null, null, '500']}
+        lineHeight={['300', null, null, '500']}
+        fontWeight="normal"
+      >
+        {subtitle}
+      </Box>
+    );
+  }
 
-const Breadcrumb = ({ content, ...rest }) => (
-  <UnstyledLink {...rest}>
-    <ChevronLeft size={27} />
-    {content}
-  </UnstyledLink>
-);
+  return <div>{subtitle}</div>;
+}
 
-class Page extends Component {
-  static displayName = 'Page';
+// TODO verify this with Popover and Actionlist tickets
+function SecondaryActions({ actions = [], hasPrimaryAction }) {
+  const visibleActions = React.useMemo(() => filterByVisible(actions), [actions]);
 
-  static defaultProps = {
-    empty: {}
+  if (!visibleActions.length) {
+    return null;
   }
 
-  static propTypes = {
-    /**
-     * The Page display title
-     */
-    title: PropTypes.node,
-
-    /**
-     * Subtitle that appears to the right of the title
-     */
-    subtitle: PropTypes.node,
-
-    /**
-      * Main cta. Most button props will work in here.
-      * e.g. { content: 'button label', onClick: callback() }
-      */
-    primaryAction: PropTypes.shape({
-      content: PropTypes.node.isRequired
-    }),
+  return (
+    <Popover
```

Comment:
Won't this need some aria property to connect it with the button?

Reply:
I think so, but can address it in the popover ticket. I believe all it needs is an `id` and `describedby`, but probably also a focus handler